src/bindist/fdppauto.bat

```diff
@@ -10,5 +10,5 @@ echo Welcome to dosemu2!
 system -s DOSEMU_VERSION
 echo     Build %DOSEMU_VERSION%
 call exechlp.bat -ep
-if exist %DOSEMUDRV%:\dosemu\dosrc.d\sugfdusr.bat call %DOSEMUDRV%:\dosemu\dosrc.d\sugfdusr.bat
+if exist %DOSEMUDRV%:\dosemu\dosrc.d for %b in (%DOSEMUDRV%:\dosemu\dosrc.d\*.bat) do call %b
```

Comment:
So with one % I suppose it will work
with other command.com's, but you
say its not the case?